products/structured-data/integration-efolder.md

```diff
@@ -20,6 +20,7 @@ The goal of the eFolder API is to allow unstructured documents to be directly up
 * related docs on Central Mail:
     https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/services/vba_documents/UnderstandingDocumentProcessing.md
 * https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Products/Identity/eFolder
```

Comment:
Just checking, in case this vets.gov-team link wasn't intended to keep.

Reply:
I think I left that one in; I actually don't have access to vets.gov-team yet - if you can see it and see that the contents are the same you could remove it.